src/utils/hooks/useSubmitReview.ts

```diff
@@ -33,8 +33,6 @@ const useSubmitReview: UseSubmitReview = (reviewId, reload) => {
 
     const reviewResponsesPatches = reviewResponses.map(({ thisReviewLatestResponse }) => ({
       patch: {
-        decision: thisReviewLatestResponse?.decision,
-        comment: thisReviewLatestResponse?.comment,
```

Comment:
Is this the fix for the incorrectly trimmed review responses? If so, why are we *removing* these? (I'm sure there's a good reason, I'm just a bit confused)